tardis/plasma/base.py

```diff
@@ -258,10 +260,35 @@ def remove_hidden_properties(self, print_graph):
                 print_graph.remove_node(str(item.name))
         return print_graph
 
-class StandardPlasma(BasePlasma):
+    def to_hdf(self, path_or_buf, path='', close_hdf=True, filter=None):
```

Comment:
I withdraw my "complicated beast" 😉 and simply say. Let's take it out for now and finish all the other `to_hdf` methods to see what we need and what we don't need.